spec/dummy/db/schema.rb

```diff
@@ -8,43 +9,42 @@
 # from scratch. The latter is a flawed and unsustainable approach (the more migrations
 # you'll amass, the slower it'll run and the greater likelihood for issues).
 #
-# It's strongly recommended that you check this file into your version control system.
+# It's strongly recommended to check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20140113233821) do
+ActiveRecord::Schema.define(:version => 20140113233821) do
```

Comment:
Minor point.. but you shouldn't have to change the hash syntax since the new style was introduced in 1.9